frontend/src/scenes/project/Settings/GroupAnalytics.tsx

```diff
@@ -68,6 +69,15 @@ export function GroupAnalytics(): JSX.Element | null {
                 This project has access to group analytics. Below you can configure how various group types are
                 displayed throughout the app.
             </p>
+
+            {groupsAccessStatus !== GroupsAccessStatus.HasGroupTypes && (
+                <AlertMessage type="info" className="mb-4">
+                    Group types will show up here after you send your first event associated with a group. Take a look
+                    at <a href="https://posthog.com/manual/group-analytics">this guide</a> for more information on
+                    getting started.
```

Comment:
Should use a `Link` element here instead of `a`

```suggestion
                    at <Link to={`https://posthog.com/manual/group-analytics`}>this guide</Link>
                    for more information on getting started.
```